src/tools/linter-rules.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Linter rules
+description: Details about the Dart Linter and the lint rules it provides.
+---
+
+The Dart Linter uses the analyzer to identify and report
+on "lints" found within Dart code. Linting is performed
+by the Dart analysis server through your IDE
+or with the [`dart analyze`](/tools/dart-analyze) command.
+
+This page outlines all currently existing rules,
+their use cases, what triggers them, as well as potential fixes.
+
+The rules are organized into rule groups
+which identify their type of functionality.
+
+- **[Errors](#error-rules)**
```

Comment:
One-item bulleted lists (such as the sublists here) are generally a no-no. We could just merge the sublists into the previous list item, or we could use a `<dl>` instead. Something like this:

* **Errors:** Possible errors or mistakes in your code.
* **Style:** Matters of style, largely derived from the Dart style guide.
* **Pub:** Possible issues with Pub setup.

Or this (and today I learned that GitHub-flavored Markdown doesn't seem to support definition lists!):

Errors
: Possible errors or mistakes in your code.

Style
: Matters of style, largely derived from the Dart style guide.

Pub
: Possible issues with Pub setup.

Reply:
I went with the Kramdown definition list, seems to work fine, thanks for the suggestion!